blivet/formats/luks.py

```diff
@@ -280,6 +281,16 @@ def _create(self, **kwargs):
                         type=self.type, status=self.status)
         super(LUKS, self)._create(**kwargs)  # set up the event sync
 
+        if not self.pbkdf_args:
```

Comment:
You should also check the version of LUKS, otherwise liblockdev might fail for LUKS1.

Reply:
Fixed.